draft-ietf-quic-manageability.md

```diff
@@ -160,17 +162,24 @@ information may be exposed in QUIC packet headers:
   connection ID is implicit.
 
 - length: the length of the remaining quic packet after the length field,
-  present on
-  long headers. This field is used to implement coalesced packets during the
-  handshake (see {{coalesced}}).
+  present on long headers. This field is used to implement coalesced packets
+  during the handshake (see {{coalesced}}).
+
+- spin bit: the spin bit supports passive measurement of RTT, and is present on
+  QUIC short packet headers. See {{sec-rtt}}.
```

Comment:
add the token field (with accompanying length) from the Initial packet

Reply:
done.